types/firestore.d.ts

```diff
@@ -2474,7 +2583,7 @@ declare namespace FirebaseFirestore {
     static where(
       fieldPath: string | FieldPath,
       opStr: WhereFilterOp,
-      value: unknown
+      value: any
```

Comment:
nit: Change `any` back to `unknown` unless this change is strictly required for the main changes in this PR.